### PR TITLE
MuhammedKalkan.OpenLens version 6.4.15

### PR DESCRIPTION
**Pull request by brotich**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102237)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation failed with:
> Installation failed with exit code -1978335146

File: manifests/m/MuhammedKalkan/OpenLens/6.4.15/MuhammedKalkan.OpenLens.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.4.15/OpenLens-6.4.15.exe
+  InstallerSha256: 61080DCDD740DF0A476576A56EC19666F32D6F7121B4E22B8B03357AB5425AA5
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+  ElevationRequirement: elevationProhibited
```
Comment (brotich) [inline manifests/m/MuhammedKalkan/OpenLens/6.4.15/MuhammedKalkan.OpenLens.installer.yaml:20]:
```suggestion
  ElevationRequirement: elevationProhibited
```

File: manifests/m/MuhammedKalkan/OpenLens/6.4.15/MuhammedKalkan.OpenLens.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.4.15/OpenLens-6.4.15.exe
+  InstallerSha256: 61080DCDD740DF0A476576A56EC19666F32D6F7121B4E22B8B03357AB5425AA5
+  InstallerSwitches:
+    Custom: /ALLUSERS
+  ElevationRequirement: elevationRequired
```
Comment (brotich) [inline manifests/m/MuhammedKalkan/OpenLens/6.4.15/MuhammedKalkan.OpenLens.installer.yaml:28]:
```suggestion
```

**Comment by brotich** [comment]
> I'm not able to find the cause for this error. It installs and runs normally on a Windows 11 VM.

I had added `ElevationRequirement` to the installer, let me remove it and see what happens

**Comment by brotich** [comment]
> 

Looks like this is related to the https://github.com/microsoft/winget-pkgs/issues/72224

**Review by stephengillie — APPROVED** [review]

**Pull request merged**